client/my-sites/themes/controller.js

```diff
@@ -68,6 +69,16 @@ function makeElement( ThemesComponent, Head, store, props ) {
 	);
 };
 
+// Where do we put this? It's client/server-agnostic, so not in client/controller,
+// which requires ReactDom... Maybe in lib/react-helpers?
+const makeLayout = ( store, primary, secondary, tertiary ) => (
```

Comment:
Yeah, pretty sure now that `makeLayout` should be middleware -> 9e7cf3ae967fa61d9ca3c5c300875bda666e8f8b.